### ISSUE TITLE
Carousel plugin - numberOfPositions?

### ISSUE THREAD

**Issue by dfector** [malsup/cycle2#45]
Awesome plugin!  I've been a long time fan since version 1.

Is there a preferred method to specify the number of slides to advance in a carousel?

I tried overriding the advanceSlide function in order to create functionality to skip forward and backward over multiple images in my carousel.  But things don't seem to work as expected.

**Comment by malsup** [comment]
Yeah, that's tricky.  Right now it's designed to advance a single slide at at time.  But of course you could have multiple images in a slide.

**Comment by dfector** [comment]
Ahhhh... clever!  I hadn't even thought of that.  So the simplest solution is to wrap groups of slides.  Perfect.

Thanks again for creating such an amazing plugin.

**Comment by AndrewMcMillan** [comment]
Hi Mike! Thanks for a great plugin I've used it a ton of times on various sites, and Cycle 2 is way better than the first awesome plugin! We have a need to rotate several at a time on a site, ranging from 3 -5 slides depending on the use case and page locations. Are there any plans on adding this feature in a dynamic fashion to the carousel so that it's configurable? Thanks again!

**Comment by mannyvergel** [comment]
I have a requirement for this functionality. Any workaround aside from having multiple images in one slide (since the desired effect won't be the same)? I'm thinking it's the same as clicking specific pages, maybe we can override the function and just manually control the prev / next actions?

**Comment by malsup** [comment]
This is a feature I'd like to add when time allows.

**Comment by paulcredmond** [comment]
Would also love this functionality.

**Comment by jamesmehorter** [comment]
+1 - Multiple images in one slide doesn't really work as expected unless you can sacrifice your image count. E.g. Say you have visible: 3, and hence want the prev/next to cycle through 3 images at a time, you could end up with something like.. 

```
<div>
    <img>
    <img>
    <img>
</div>
<div>
    <img>
    <img>
    <img>
</div>
<div>
    <img>
</div>
```

Where the last slide only contains one image (as an example). I have got this working by only pulling a slide count from my db in intervals of 3.. i.e. I only pull 6 images, not 7. 

However... then I might as well use scrollHorz instead. Not sure what defines a 'carousel'. _shrug_

**Comment by pigmeu** [comment]
Also hoping for this functionality to come along soon...
